examples/restaurant_demo/restaurant_demo.ipynb

```diff
@@ -0,0 +1,3257 @@
+{
```

Comment:
<p>Probably explain in detail what this function is supposed to do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PyDP/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>convert the visitor time to 24 hrs format from 12 hrs format as commented. For partitions contributed to. Would you like me to explain the steps? </p>

Reply:
yes, what would be great for somone going through the code.